smartling-projects-api/src/test/java/com/smartling/api/projects/v2/ProjectsApiFactoryTest.java

```diff
@@ -0,0 +1,114 @@
+package com.smartling.api.projects.v2;
+
+import com.smartling.api.projects.v2.pto.LocaleDetailsPTO;
+import com.smartling.api.projects.v2.pto.ProjectDetailsPTO;
+import com.smartling.api.v2.client.ClientConfiguration;
+import com.smartling.api.v2.client.DefaultClientConfiguration;
+import com.smartling.api.v2.client.auth.BearerAuthStaticTokenFilter;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.apache.http.HttpStatus;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.ws.rs.core.HttpHeaders;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class ProjectsApiFactoryTest
+{
+    private static final String SUCCESS_RESPONSE_ENVELOPE = "{ \"response\": { \"code\": \"SUCCESS\", \"data\": %s } })";
+
+    private static final String GET_PROJECTS_DETAILS_DATA_JSON =
```

Comment:
is it possible to move JSON to a resource file? It's unreadable in code as for me